x-pack/plugins/alerts/server/alert_type_registry.ts

```diff
@@ -71,18 +82,23 @@ export class AlertTypeRegistry {
       );
     }
     alertType.actionVariables = normalizedActionVariables(alertType.actionVariables);
-    this.alertTypes.set(alertIdSchema.validate(alertType.id), { ...alertType });
+    this.alertTypes.set(alertIdSchema.validate(alertType.id), { ...alertType } as AlertType);
     this.taskManager.registerTaskDefinitions({
       [`alerting:${alertType.id}`]: {
         title: alertType.name,
         type: `alerting:${alertType.id}`,
         createTaskRunner: (context: RunContext) =>
-          this.taskRunnerFactory.create(alertType, context),
+          this.taskRunnerFactory.create({ ...alertType } as AlertType, context),
```

Comment:
`{ ...alertType }` is a "while I'm here" for shallow clones that was missed in https://github.com/elastic/kibana/pull/65309.